src/Utils.php

```diff
@@ -21,10 +21,10 @@ final class Utils
      */
     public static function describeType($input): string
     {
-        switch (\gettype($input)) {
-            case 'object':
+        switch (true) {
+            case \is_object($input):
                 return 'object(' . \get_class($input) . ')';
-            case 'array':
+            case \is_array($input):
```

Comment:
PHPStan cannot understand this as `$input` still is `mixed` despite calling `gettype()`, so by calling the introspection functions allows PHPStan to understand the intended type here

Reply:
please revert all changes to this file.